### PR TITLE
feat: set dark navigation bar color for reverse image screens

### PR DESCRIPTION
**Pull request by dimatretyak**
### Demo
| Before | After |
| ------------- | ------------- |
| before | after |

#### Demo video
https://user-images.githubusercontent.com/3513494/189325466-b9ba14a0-ec94-47d0-88ea-0a3d4ff610ac.mp4

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- set dark navigation bar color for reverse image screens - dimatretyak

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/ReverseImage/ReverseImage.tsx [hunk 1]
```diff
@@ -16,6 +20,8 @@ export const ReverseImage = () => {
 
     if (Platform.OS === "android") {
       StatusBar.setBackgroundColor("transparent")
+      ArtsyNativeModule.setNavigationBarColor("#000000")
+      ArtsyNativeModule.setAppLightContrast(true)
```
Comment (gkartalis) [inline src/app/Scenes/ReverseImage/ReverseImage.tsx:24]: **question:** after this change, if you navigate away from this screen, does the bar go back to default again?

**Review by gkartalis — APPROVED** [review]

File: src/app/Scenes/ReverseImage/ReverseImage.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Scenes/ReverseImage/ReverseImage.tsx:24]: like for example, if you press the bottom tab and try to navigate to profile

Comment (dimatretyak) [reply src/app/Scenes/ReverseImage/ReverseImage.tsx:24]: yep. you can see it on the demo video

Comment (dimatretyak) [reply src/app/Scenes/ReverseImage/ReverseImage.tsx:24]: > like for example, if you press the bottom tab and try to navigate to profile
if I understood you correctly, then the bottom tab is not displayed on the reverse image screens

Comment (gkartalis) [reply src/app/Scenes/ReverseImage/ReverseImage.tsx:24]: perfect! 

⛴⛴⛴⛴⛴
⛴⛴⛴⛴⛴
⛴⛴⛴⛴⛴ 
⛴⛴⛴⛴⛴ 
⛴⛴⛴⛴⛴

**Pull request merged**